language-design.md

```diff
@@ -38,7 +96,23 @@ How do the themes of _Growing a Language_ relate to the "sound lab" we did this
 
 **Response**
 
-
+In _Growing a Language_, Steele conveys that well-designed languages should
+start from a simple base and optimize for growth. Building out a fully packed
+language from the start does not follow good design principles. Another major
+point he brought up is that not every possible feature should be added to a
+language. In our sound lab, we tried to improve the fluency of an existing
+sound library. The existing code allowed for very separate processes and did not
+allow for users to really implement their desired behaviors easily. For example,
+reversing and slowing down a single piece of audio was not easy. Our goal was
+to improve the fluency such that a user could use the existing language to build
+out their own features, and eventually _grow_ the language appropriately. The
+library did not really allow for growth.  A bunch of separate methods that could
+not be linked did not optimize for growth, so we tried to improve it, as Steele
+would have probably done himself.  Furthermore, if we had decided to fully 
+implement the sound lab, we would likely have talked to people that may use
+the program to transform music. Since our design decisions and use cases 
+are based off of a programmer’s thought process, these decisions may not
+be optimal for people actually using the sound library.
 
```

Comment:
Good point about if we were _actually_ going to create a sound API, we would interview potential users to see what functionality they wanted.  Perhaps, since the use case was CS5 Labs, I should give the API more credit since new programmers haven't yet been introduced to classes or other features that would improve the usability.  I'm curious about specifically how you edited the sound API to allow for growth.  What methods did you include / exclude?  What about your design improved fluency?  How, as a user of your code, could I grow the API more easily than before?